tools/c7n_org/scripts/orgaccounts.py

```diff
@@ -72,6 +73,24 @@ def main(role, ou, assume, profile, output, region):
         file=output)
 
 
+def get_org_client(assume, profile):
+    if assume:
```

Comment:
fwiw, we have this abstraction and handling already in `c7n.credentials.SessionFactory`